From 0f7abc7f08e088bbc9448b3a77def7c6da61c507 Mon Sep 17 00:00:00 2001
From: Michal Suchanek <msuchanek@suse.de>
Date: Wed, 27 May 2020 09:52:07 -0700
Subject: [PATCH] ofpathname: Fix udevadm location

Upstream: pending - expected 1.3.8
Git-commit: 0f7abc7f08e088bbc9448b3a77def7c6da61c507

The canonical location for udevadm is /usr/bin.

Signed-off-by: Michal Suchanek <msuchanek@suse.de>
Signed-off-by: Tyrel Datwyler <tyreld@linux.ibm.com>
---
 scripts/ofpathname | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/ofpathname b/scripts/ofpathname
index bd61ee04bc14..0fb00ade8167 100755
--- a/scripts/ofpathname
+++ b/scripts/ofpathname
@@ -349,7 +349,7 @@ is_pata_dev()
     local sysfs_dir
     local udev_path
     local udevinfo="/usr/bin/udevinfo"
-    local udevadm="/sbin/udevadm"
+    local udevadm="/usr/bin/udevadm"
 
     if [[ -a $udevadm ]]; then
         udev_path=`$udevadm info --query=path --name=$DEVNAME`
-- 
2.26.2

